Obsidian.API/_Types/CraftingType.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Obsidian.API
+{
+    public enum CraftingType
+    {
+        CraftingShaped,
+        CraftingShapeless,
+
+        CraftingSpecialArmordye,
```

Comment:
I feel this naming is inconsistant
> CraftingSpecialArmor**D**ye

Reply:
Its the names used by minecraft itself just converted from Identifiers to PascalCase